docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+title: Run Tests on LambdaTest with CircleCI
+permalink: /documentation/guides/continuous-integration/circleci-and-lambdatest.html
+---
+# Run Tests on LambdaTest with CircleCI
+
+This topic describes how to integrate TestCafe into the [CircleCI](https://circleci.com/) build process and run tests in the [LambdaTest](https://www.lambdatest.com) cloud testing service.
+
+## Prerequisites
+
+In this tutorial, we will fetch tests from a GitHub repository: [testcafe-ci-demo](https://github.com/DevExpress-Examples/testcafe-ci-demo). Fork this repository before we start.
+
+> If you use a different version control system, make sure CircleCI supports it. To learn which systems it supports, refer to the [CircleCI Integrations](https://circleci.com/integrations/) page.
```

Comment:
```suggestion
> If you use a different control system version, make sure CircleCI supports it. For information on which systems you can use, refer to the [CircleCI Integrations](https://circleci.com/integrations/) page.
```

Reply:
I didn't mean `control system version`, it's `version control system` (or `version-control system`), like Git:
https://en.wikipedia.org/wiki/Git

Reply:
Ok